test/iden3comm/domain/use_cases/get_iden3comm_proofs_use_case_test.dart

```diff
@@ -95,6 +98,9 @@ main() {
 
     when(generateProofUseCase.execute(param: anyNamed('param'))).thenAnswer(
         (realInvocation) => Future.value(Iden3commMocks.jwzSdProof));
+
+    when(getIdentityUseCase.execute(param: anyNamed('param'))).thenAnswer(
+        (realInvocation) => Future.value(IdentityMocks.privateIdentity));
```

Comment:
Missing the params capture check of this one.

Reply:
done